api/src/main/java/graphql/nadel/schema/UnderlyingWiringFactory.java

```diff
@@ -39,38 +32,17 @@ public boolean providesTypeResolver(InterfaceWiringEnvironment environment) {
 
     @Override
     public TypeResolver getTypeResolver(InterfaceWiringEnvironment environment) {
-        return underScoreTypeNameResolver();
+        return delegateWiringFactory.getTypeResolver(environment);
```

Comment:
Tbh idk what this class really does. I think we can just delete it? After removing this code it seems to just delegate everything to `delegateWiringFactory`…